src/integrationTest/java/uk/gov/hmcts/reform/cpo/utils/CasePaymentOrderEntityGenerator.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.cpo.utils;
+
+import org.apache.commons.lang3.RandomUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.cpo.data.CasePaymentOrderEntity;
+import uk.gov.hmcts.reform.cpo.domain.CasePaymentOrder;
+import uk.gov.hmcts.reform.cpo.repository.CasePaymentOrdersJpaRepository;
+import uk.gov.hmcts.reform.cpo.service.mapper.CasePaymentOrderMapper;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CasePaymentOrderEntityGenerator {
+
+    @Autowired
+    private CasePaymentOrdersJpaRepository casePaymentOrdersJpaRepository;
+
+    @Autowired
+    private CasePaymentOrderMapper casePaymentOrderMapper;
+
+    @Autowired
+    private UIDService uidService;
+
+    public List<CasePaymentOrderEntity> generateAndSaveEntities(int numberToGenerate) {
+
```

Comment:
is generateAndSaveEntities() for testing only or it is part of a requirement ?

Reply:
It's just a common utility class used by tests, that's why i've put it in the `integrationTest` package